backend/vendor/assets/javascripts/jquery.jstree/themes/apple/style.scss

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
Whoops. The rebase snuck that back in. This is fixed and I gave this a more careful read.